index.html

```diff
@@ -0,0 +1,557 @@
+          <h1 class="footer__title">KMarketer</h1>
+          <div class="footer__icons-container">
+            <img
+              class="footer__icon"
+              src="./assets/images/Vector-fb-footer.png"
+            />
+            <img
+              class="footer__icon"
+              src="./assets/images/Vector-insta-footer.png"
+            />
+            <img
+              class="footer__icon"
+              src="./assets/images/Vector-twitter-footer.png"
+            />
+            <img
+              class="footer__icon"
+              src="./assets/images/Vector-youtube-footer.png"
+            />
+          </div>
+        </div>
+        <div class="footer__menu">
+          <div class="footer__card">
+            <h2 class="footer__card-title">Company Info</h2>
+            <ul class="footer__card-menu item item--lighter">
+              <li class="footer__menu-item">About Us</li>
+              <li class="footer__menu-item">Carrier</li>
+              <li class="footer__menu-item">We are hiring</li>
+              <li class="footer__menu-item">Blog</li>
+            </ul>
+          </div>
+
+          <div class="footer__card">
+            <h2 class="footer__card-title">Legal</h2>
+            <ul class="footer__card-menu item item--lighter">
+              <li class="footer__menu-item">About Us</li>
+              <li class="footer__menu-item">Carrier</li>
+              <li class="footer__menu-item">We are hiring</li>
+              <li class="footer__menu-item">Blog</li>
+            </ul>
+          </div>
+
+          <div class="footer__card">
+            <h2 class="footer__card-title">Features</h2>
+            <ul class="footer__card-menu item item--lighter">
+              <li class="footer__menu-item">Business Marketing</li>
+              <li class="footer__menu-item">User Analytic</li>
+              <li class="footer__menu-item">Live Chat</li>
+              <li class="footer__menu-item">Unlimited Support</li>
+            </ul>
+          </div>
+
+          <div class="footer__card">
+            <h2 class="footer__card-title">Resources</h2>
+            <ul class="footer__card-menu item item--lighter">
+              <li class="footer__menu-item">IOS & Android</li>
+              <li class="footer__menu-item">Watch a Demo</li>
+              <li class="footer__menu-item">Customers</li>
+              <li class="footer__menu-item">API</li>
+            </ul>
+          </div>
+
+          <div class="footer__card footer__card--span-full">
+            <h2 class="footer__card-title">Get In Touch</h2>
+            <div class="footer__card-form">
+              <input
+                type="email"
+                placeholder="Your Email"
+                class="footer__card-input"
+              />
+              <button class="footer__btn footer__card-btn">Subscribe</button>
+            </div>
+          </div>
+        </div>
+      </footer>
+
+      <div class="last-footer-container">
```

Comment:
improve the naming, may be `last-footer` only
and make second block `last-footer__container`